src/track/libheaptrack.cpp

```diff
@@ -607,6 +717,10 @@ class HeapTrack
                         HeapTrack heaptrack([&] { return !stopTimerThread.load(); });
                         heaptrack.writeTimestamp();
                         heaptrack.writeRSS();
+                        counter = (counter + 1) % 100;
+                        if (counter == 0) {
+                            heaptrack.writeInRam();
```

Comment:
this means that the data is potentially obsolete when the application ends. at the very least you should try to write the ram data when the app is closing down, no?